### PR TITLE
Fix #27: don't "hang" on empty stdin

### LINKED ISSUE #27: gntagger with no options hangs (mac OS)
```
when I run `gntagger` (with no options)
   then the executable "hangs"
      and I do not get the shell cursor back
   I would prefer to see `gntagger -h` in this case
```

### PR DESCRIPTION
**Pull request by alexander-myltsev**
fix #27 

There are more ways to solve it:
* https://stackoverflow.com/questions/22744443/check-if-there-is-something-to-read-on-stdin-in-golang 
* https://stackoverflow.com/questions/22563616/determine-if-stdin-has-data-with-go

### CODE REVIEW

**Review by dimus — DISMISSED** [review]

**Comment by dimus** [comment]
Did not work for me with file, I am trying an approach from https://stackoverflow.com/a/26567513/23080
@alexander-myltsev can you check if it works for you too?

File: cmd/gntagger/main.go [hunk 1]
```diff
@@ -35,23 +35,16 @@ func main() {
...
-		flag.Usage()
-		return
-	}
-
 	switch flag.NArg() {
 	case 0:
-		data, err = ioutil.ReadAll(os.Stdin)
-		if err != nil {
-			log.Panic(err)
+		if ok := checkStdin(); ok {
```
Comment (alexander-myltsev) [inline cmd/gntagger/main.go:40]: This is redundant `ok` introduction

File: cmd/gntagger/main.go [hunk 2]
```diff
@@ -77,6 +70,15 @@ func main() {
 	InitGUI(txt, names)
 }
 
+func checkStdin() bool {
```
Comment (alexander-myltsev) [inline cmd/gntagger/main.go:73]: `checkIfStdin`?

**Comment by alexander-myltsev** [comment]
It works for me.

**Pull request closed without merging**